src/plugins/intel_cpu/src/nodes/space_to_batch.cpp

```diff
@@ -169,6 +166,9 @@ void SpaceToBatch::SpaceToBatchKernel() {
         std::vector<size_t> indxStart(2, 0);
         std::vector<size_t> indxEnd(2, 0);
         parallel_it_init(start, indxStart[0], inShape5D[0], indxStart[1], channels);
+        if (start >= workAmount) {
```

Comment:
As I can see, We usually use the condition `start >= end`. Maybe we should use the same way. What do you think?

Reply:
Agree. Modified